src/components/10-atoms/radio/index.js

```diff
@@ -0,0 +1,251 @@
+import { html, svg } from 'lit-element';
+import NoShadowDOM from '../../../utils/no-shadow';
+import defaultName from '../../../utils/create-ref-id';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+// module globals
+const radioButtonGroup = {};
+const selectedRadioButton = {};
+const maxWidth = {};
+
+const WIDTH_SLACK = 5; // px
+const DOM_PAINT_TIME = 1000 * (2 / 60); // ms - 2 frames at 60 fps
```

Comment:
I would use `1` cause it is not possible that JS works parallel. Code inside setTimeout is ALWAYS executed AFTER current executing code is finished, also when current code (including painting) takes longer.

Reply:
For me is overeng this and actually lowers the felt response time more than necessary

Reply:
Changed to 1